e2e-tests/ipset-build-test.yaml

```diff
@@ -0,0 +1,90 @@
+package:
+  name: ipset
+  version: "7.22"
+  epoch: 3
+  description: Manage Linux IP sets
+  copyright:
+    - license: GPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libmnl-dev
+      - libtool
+      - linux-headers
+      - pkgconf-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://git.netfilter.org/ipset
+      tag: v${{package.version}}
+      expected-commit: a50abde9c959be364782c01c61429a951454f5ef
+      depth: "-1"
+
+  - runs: |
+      ./autogen.sh
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --build=${{host.triplet.gnu}} \
+        --host=${{host.triplet.gnu}} \
+        --with-kmod=no \
+        --prefix=/usr
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
```

Comment:
The subpackage pipelines for this package are somewhat interesting. I wanted to make sure something other than symlinks was being tested with the new changes.